docs/software-platforms/aws/working-with-s3-buckets.md

```diff
@@ -40,4 +40,69 @@ You can then use `$OPENSCPCA_RESULTS_BUCKET` (or, `${OPENSCPCA_RESULTS_BUCKET}`)
 
 ## Syncing your results to S3
 
-_Coming up in the next PR._
+We have written a script to help you sync your results to your S3 bucket, stored in [`scripts/sync-results.py`](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/scripts/sync-results.py).
+For this script to work, you need to be [logged into the AWS account profile you use for contributing to OpenScPCA](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
+
+This script syncs the contents of a given analysis module's `results` and `plots` directories from your computer to S3.
+It does not also sync S3 contents back to your computer.
+Your files will be synced to a bucket called `s3://{your bucket name}/{analysis module name}`, where:
```

Comment:
I don't love "location" here, but I want to make the distinction between a bucket, which is just the overall place, and files/"directories" within the bucket. 
```suggestion
Your files will be synced to the S3 location `s3://{your bucket name}/{analysis module name}`, where:
```